package.json

```diff
@@ -65,7 +65,11 @@
     "@hot-loader/react-dom": "^16.8.6",
     "@testing-library/react": "^9.0.1",
     "@testing-library/react-hooks": "^2.0.1",
+    "@types/jest": "^24.9.0",
```

Comment:
Would make sense to add type definitions for other libraries as well, e.g. `@types/react-dom`

Reply:
Yeah makes sense. Should add them here or should I leave it to whenever they will be needed, eg. switching the first React component to TS?

Reply:
I think we should wait until we need them, but very good point @HerrEmil